### ISSUE TITLE
Traffic to localhost isn't proxied

### ISSUE THREAD

**Issue by ngregoire** [nccgroup/autochrome#24]
Hello,

by default, traffic to localhost, 127.0.0.1, ... isn't proxied.
That can be fixed by adding the following line to the `chrome` script:

    "--proxy-bypass-list=<-loopback>",

Details at https://chromium.googlesource.com/chromium/src/+/master/net/docs/proxy.md#Bypass-rule_Subtract-implicit-rules

**Comment by peter-h-ncc** [comment]
Does this break functionality such as accessing `http://burp/` ?

**Comment by ngregoire** [comment]
No, it doesn't. Only explicitly accessing `http://127.0.0.1/` is affected
